### PR TITLE
[10.0][MIG] partner_sector

### PR DESCRIPTION
**Pull request by luismontalba**
Partner Sector
==============
This module adds the concept of economic sector for partners.

Configuration
-----------------
To manage sectors, you need to:
* Go to *Sales > Configuration > Contacts > Sectors*.

Usage
--------

To use this module, you need to:

* Go to any partner's form.

Only companies have sectors.

cc @Tecnativa

### CODE REVIEW

**Review by yajo — CHANGES_REQUESTED** [review]
Great! Just squash the translation commits please.

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍

**Review by yajo — APPROVED** [review]

**Comment by yajo** [comment]
OK, not blocking just for the squash

File: partner_sector/views/res_partner_sector_view.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
     <menuitem action="res_partner_sector_action"
               id="menu_res_partner_sector"
               name="Sectors"
-              parent="base.menu_config_address_book"
+              parent="sales_team.menu_sale_config"
```
Comment (chienandalu) [inline partner_sector/views/res_partner_sector_view.xml]: Isn't it better to keep it on sales_team.menu_sale_config?

File: partner_sector/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 Antiun Ingenieria S.L. - Javier Iniesta
+# Copyright 2016-2017 Tecnativa S.L. - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Partner Sector",
+    "summary": "Add partner sectors",
+    "version": "10.0.1.0.1",
```
Comment (pedrobaeza) [inline partner_sector/__manifest__.py]: 10.0.1.0.0

File: partner_sector/models/res_partner_sector.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    _parent_store = True
+    _description = "Sector"
+
+    name = fields.Char(required=True, translate=True)
+    parent_id = fields.Many2one(comodel_name='res.partner.sector',
+                                ondelete='restrict')
+    child_ids = fields.One2many(comodel_name='res.partner.sector',
+                                inverse_name='parent_id',
+                                string="Children")
+    # parent_left = fields.Integer('Parent Left', select=True)
```
Comment (pedrobaeza) [inline partner_sector/models/res_partner_sector.py]: Please remove these commented lines

File: partner_sector/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    "category": "Customer Relationship Management",
+    "website": "http://www.tecnativa.com",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "sale",
```
Comment (pedrobaeza) [inline partner_sector/__manifest__.py]: Depends on sales_team, not sale.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
The Travis error is also present upstream. I'm going to check.

File: partner_sector/views/res_partner_sector_view.xml (hunk 1 above)
Comment (yajo) [reply partner_sector/views/res_partner_sector_view.xml]: Isn't just that what it is doing?

Comment (chienandalu) [reply partner_sector/views/res_partner_sector_view.xml]: Sorry, I meant sales_team.menu_config_address_book :smile:

Comment (luismontalba) [reply partner_sector/views/res_partner_sector_view.xml]: Yes, I agree it's a better location ...

**Review by chienandalu — APPROVED** [review]
:smiley: :+1:

**Comment by yajo** [comment]
happy, @pedrobaeza?

**Comment by yajo** [comment]
Travis error seems to be ~~https://github.com/OCA/maintainer-quality-tools/issues/448~~ #434.

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
As the error comes from another module, I merge.

**Pull request merged**